src/main/kotlin/com/evalify/evalifybackend/questions/domain/FillUp.kt

```
@@ -0,0 +1,4 @@
+package com.evalify.evalifybackend.questions.domain
+
+class FillUp {
+}
```

detekt (1.23.8)

[warning] The class or object FillUp is empty.

(detekt.empty-blocks.EmptyClassBlock)